package.json

```diff
@@ -35,6 +35,7 @@
     "@types/source-map": "^0.5.1",
     "@types/source-map-support": "^0.2.27",
     "chai": "^3.5.0",
+    "chai-diff": "^1.0.1",
```

Comment:
This is good, but FWIW in my experience it's easier to just let the thing write out new goldens and examine the output with 'git diff'.

Reply:
When I'm interactively tweaking the emit, I find it much easier to see a good diff there than re-gen the goldens. That also runs a high risk of missing a change. But I understand workflows are different.